src/allmydata/scripts/common_http.py

```diff
@@ -85,7 +77,7 @@ def do_http(method, url, body=b""):
         return BadResponse(url, err)
 
     while True:
-        data = body.read(8192)
+        data = body.read(65536)
```

Comment:
This probably wants to be spelled "blocksize" (instead of different magic number)..?